Kvent/views.py

```diff
@@ -50,9 +50,11 @@ def create_event(request):
             location = form.data.get('location')
             short_description = form.data.get('short_description')
             long_description = form.data.get('long_description')
+            arrange_time = form.data.get('arrange_time')
+            print(arrange_time)
```

Comment:
delete the print statement that you use to test the code

Reply:
I already fix this part.